blocks/blocks.go

```diff
@@ -11,16 +12,20 @@ import (
 	mh "gx/ipfs/QmbZ6Cee2uHjG7hf19qLHppgKDRtaG4CVtMzdmK9VCVqLu/go-multihash"
 )
 
-var ErrWrongHash = errors.New("data did not match given hash!")
+// ErrWrongHash is returned when the Cid of a block is not the expected
+// according to the contents. It is currently used only when debugging.
```

Comment:
That uses `ErrHashMismatch = errors.New("block in storage has different hash than requested")` in the blockstore module...